.circleci/config.yml

```diff
@@ -391,21 +403,18 @@ jobs:
           avd-name: TwilioVoiceReactNativeReferenceApp_AVD
           install: true
           system-image: system-images;android-29;google_apis;x86_64
+          additional-args: --device pixel_4
       - android/start-emulator:
           avd-name: TwilioVoiceReactNativeReferenceApp_AVD
           memory: 8192
           post-emulator-launch-assemble-command: ""
           restore-gradle-cache-post-emulator-launch: false
-          wait-for-emulator: false
```

Comment:
default `wait-for-emulator` is true, I sometimes see detox try to test when the emulator is not ready.

Reply:
I recall that I had to set this to false because the orb had a bug where this command would sometimes non-deterministically fail.

Reply:
Are you not seeing any such failures in your CI pipelines @kpchoy?

Reply:
nope I don't see that, I was seeing the detox try to run a test when the emulator is not ready. The orb seems to work in default configuration for my runs

Reply:
@mhuynh5757 I do often see the app sometimes just crashing https://app.circleci.com/pipelines/github/twilio/twilio-voice-react-native-app/853/workflows/9d6cfb89-e223-4de0-b8a2-d958f9c8518e/jobs/5968?invite=true#step-124-5699_47 . When I look at the error on detox issues, this seems to be common. Detox tests in emulators can be flakey, and its recommended to use firebase instead.